axum-macros/src/lib.rs

```diff
@@ -385,6 +386,12 @@ pub fn debug_handler(_attr: TokenStream, input: TokenStream) -> TokenStream {
     return expand_attr_with(_attr, input, debug_handler::expand);
 }
 
+/// TODO
+#[proc_macro_derive(TypedPath, attributes(typed_path))]
+pub fn derive_typed_path(input: TokenStream) -> TokenStream {
```

Comment:
Hm, does it really make sense for this to be in `axum-macros`? I think it would be more sensible to create a separate proc-macro crate for it and have all the new functionality plus a re-export of this macro in `axum-extra` behind a feature flag.

Reply:
I would like to not have too many `axum-*` crates and it doesn't add any new deps. However re-exporting it from axum-extra is a good idea, as an optional feature.